tests/Unit/DataStructures/Test_Normalize.cpp

```diff
@@ -0,0 +1,117 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/Tensor/Expressions/Normalize.hpp"
+
+TEST_CASE("Unit.DataStructures.Tensor.EuclideanNormalize",
+          "[DataStructures][Unit]") {
+  const size_t npts = 8;
+  DataVector one(npts, 1.0);
+  DataVector two(npts, 2.0);
+  DataVector three(npts, 3.0);
+  DataVector four(npts, 4.0);
+  DataVector five(npts, 5.0);
+  DataVector twelve(npts, 12.0);
+  DataVector thirteen(npts, 13.0);
```

Comment:
Additionally, in this test (and `"Unit.DataStructures.Tensor.EuclideanMagnitude"`), `thirteen` is defined but never used.